js/lesson4/tutorial.md

```diff
@@ -13,13 +13,14 @@ In this tutorial we are going to look at:
 * JSON
 * Loading API data into web pages
 * Using jQuery AJAX functionality
```

Comment:
Do we need to remove the reference to jQuery, I can no longer see it in the tutorial
```suggestion
```

Reply:
my bad, I now see it